apply-headers.sh

```diff
@@ -31,10 +46,14 @@ function applySPDXline {
     find . -type f -iname \*.$fileEnding \
     | grep -v '^./.git\|/build/\|/\.gradle/\|gradlew.bat\|sechub-cli/pkg/mod\|sechub-cli/src/mercedes-benz.com/sechub/pkg/mod/' \
     | while read file ; do
+      if ! is_excluded "$file"; then
         if ! grep -q "$SPDX_TEXT" $file ; then
             sed -i "${line}i $spdxMessage" "$file"
             echo -e "${BROWN}$file${NC} - ${LIGHT_GREEN}copyright appended.${NC}"
         fi
+      else
+         echo "Skipping $file"
```

Comment:
Good idea to introduce an exclude mechanism
```suggestion
         echo "Skipping excluded file: $file"
```